legal-api/src/legal_api/services/bootstrap.py

```diff
@@ -61,15 +61,17 @@ def delete_bootstrap(bootstrap: RegistrationBootstrap):
         return HTTPStatus.OK
 
     @staticmethod
-    def register_bootstrap(bootstrap: RegistrationBootstrap, business_name: str) \
-            -> Union[HTTPStatus, Dict]:
+    def register_bootstrap(bootstrap: RegistrationBootstrap,
+                           business_name: str,
+                           corp_type_code: str = 'TMP') -> Union[HTTPStatus, Dict]:
```

Comment:
Can we add an Enum (using the new BaseEnum) and add the corp_type for the BEN, CP, etc?
Or do we need it to fall under the act? Firm, Business, Other?

Reply:
We don't have that flexibility since the new code has to be introduced in auth-api (corp_types table), can we create an Enum against filing type (incorporationApplication, registration) so that we can extend the current implementation.